guides/source/action_text_overview.md

```diff
@@ -46,6 +46,7 @@ happens after every keystroke, and avoids the need to use execCommand at all.
 ## Installation
 
 Run `rails action_text:install` to add the Yarn package and copy over the necessary migration.
+Uncomment # gem 'image_processing', '~> 1.2' and run `bundle install`.
```

Comment:
Since Action Text uses Active Stroage to manage attachments, users also need to set up Active Storage to use Action Text (the migration file generated but users also need to be aware of config). 
I think we need an explanation on that point. The Active Storage setup is described in the Active Storage guide, so I think it's enough to have a link to that page here. 

Can you add a description of the Active Storage setup required and a link to that page?

Reply:
I changed the text to add link to  Active Storage Overview.
Thank you for your suggestion!